src/data/ACTIONS/root/PLD.ts

```diff
@@ -92,8 +92,6 @@ export const PLD = ensureActions({
 		icon: iconUrl(2506),
 		onGcd: true,
 		speedAttribute: Attribute.SKILL_SPEED,
-		cooldown: 60000,
```

Comment:
This may be in the wrong place - I haven't had the chance to get super familiar with the logic yet. It would appear on line 234 of CooldownDowntime.tsx, we calculate the number of opportunities to use the ability until the end of the fight using the action's cooldown. It seems that right now, it's expecting us to be able to use Goring Blade every 3.75s, even though it is locked behind FoF. This results in logs giving you very large expected max usages (e.g. 10/166).

I placed this comment here because I'm not sure if this change is the one that causes the behavior (as we attempt to read from gRep.cooldown when determining the step to increment currentTime with).

Reply:
Yeah, removing the cooldown will cause "use your cooldowns" to assume you can constantly use Goring Blade.  Since we now get a "Goring Blade Ready" buff, we should move Goring Blade out of the "Use Your Cooldowns" module and into a Procs tracking module instead (every time you gained Goring Blade Ready, you used it)

Reply:
Is that not already implicitly done via the Fight or Flight Analyzer?

* Fight or Flight is in the "Use Your Cooldowns" Module
* Fight or Flight analysis tells you to use Goring Blade within Fight or Flight

For now I'll remove it from the Use Your Cooldowns Module and not add an an additional analysis for it. Is that ok?

Reply:
You're correct in that Fight or Flight is in the "Use Your Cooldowns" Module (and sorry for potential ambiguity -- I use that term as the friendly name for CooldownDowntime, which you seem to have found correctly).  Last expansion, Goring Blade wasn't a direct "flip skill" after Fight or Flight, so it needed to be tracked separately within that Use Your Cooldowns module, and removing it from there is the correct way to fix the issue edreeseg noticed.

Since the Fight or Flight BuffWindow module is expecting to see Goring Blade within Fight or Flight, that may be enough to ensure people are using Goring Blade before the buff falls off.  The one potential advantage of a Procs based module is that it would provide an additional, more specific suggestion for "hey, don't let Goring Blade Ready fall off without using it", but if y'all don't feel that separate suggestion is meaningful, I don't have a strong opinion on it.

Reply:
I can implement the separate analysis if required. I think @Styrfire would be the one to make that call, though, given they're the PLD maintainer.